### PR TITLE
Feature/teku

### PR DESCRIPTION
**Pull request by unxnn**

### CODE REVIEW

File: charts/operator/templates/_helpers.tpl [hunk 1]
```diff
@@ -107,12 +107,16 @@ Validator beacon node
 - "--beacon-rpc-provider={{ $.Values.beaconChainRpcEndpoint }}"
 {{- else if eq $.Values.type "lighthouse" }}
 - "--beacon-nodes={{ $.Values.beaconChainRpcEndpoint }}"
+{{- else if eq $.Values.type "lighthouse" }}
```
Comment (tsudmi) [inline charts/operator/templates/_helpers.tpl:110]:
```suggestion
{{- else if eq $.Values.type "teku" }}
```

File: charts/operator/Chart.yaml [hunk 2]
```diff
@@ -56,3 +56,7 @@ dependencies:
     repository: https://charts.stakewise.io
     version: "0.1.0"
     condition: lighthouse.enabled
+  # - name: teku
+  #   repository: https://charts.stakewise.io
+  #   version: "1.0.0"
```
Comment (tsudmi) [inline charts/operator/Chart.yaml:61]: Shouldn't the version be here similar to others? `0.1.0`

File: charts/operator/Chart.yaml [hunk 3]
```diff
@@ -56,3 +56,7 @@ dependencies:
     repository: https://charts.stakewise.io
     version: "0.1.0"
     condition: lighthouse.enabled
+  # - name: teku
```
Comment (tsudmi) [inline charts/operator/Chart.yaml:59]: Why do we comment it out?

File: charts/operator/Chart.yaml (hunk 2 above)
Comment (unxnn) [reply charts/operator/Chart.yaml]: ok :)

File: charts/teku/values.yaml [hunk 4]
```diff
@@ -0,0 +1,350 @@
...
+    ## @param metrics.prometheusRule.namespace The namespace in which the prometheusRule will be created
+    ##
+    namespace: ""
+    ## @param metrics.prometheusRule.additionalLabels Additional labels for the prometheusRule
+    ##
+    additionalLabels: {}
+    ## @param metrics.prometheusRule.rules Custom Prometheus rules
+    ## e.g:
+    ## rules:
+    ##   - alert: PrysmBeaconNodeDown
```
Comment (tsudmi) [inline charts/teku/values.yaml:317]: Here and in other places it should be Teku

File: charts/teku/templates/servicemonitor.yaml [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+      interval: {{ .Values.metrics.serviceMonitor.interval }}
+      {{- end }}
+      {{- if .Values.metrics.serviceMonitor.scrapeTimeout }}
+      scrapeTimeout: {{ .Values.metrics.serviceMonitor.scrapeTimeout }}
+      {{- end }}
+      {{- if .Values.metrics.serviceMonitor.honorLabels }}
+      honorLabels: {{ .Values.metrics.serviceMonitor.honorLabels }}
+      {{- end }}
+      {{- if .Values.metrics.serviceMonitor.relabellings }}
+      relabelings: {{- toYaml .Values.metrics.serviceMonitor.relabellings | nindent 6 }}
```
Comment (tsudmi) [inline charts/teku/templates/servicemonitor.yaml:30]: Inconsistent variable name: `relabelings` vs `relabellings`

**Review by tsudmi — APPROVED** [review]

**Pull request merged**